includes/Integrations/Site_Kit.php

```diff
@@ -161,27 +179,40 @@ public function get_plugin_status() {
 		$is_installed        = array_key_exists( 'google-site-kit/google-site-kit.php', get_plugins() );
 		$is_active           = $this->is_plugin_active();
 		$is_analytics_active = $this->is_analytics_module_active();
+		$is_adsense_active   = $this->is_adsense_module_active();
+
+		$analytics_link = __( 'https://wordpress.org/plugins/google-site-kit/', 'web-stories' );
+		$adsense_link   = __( 'https://wordpress.org/plugins/google-site-kit/', 'web-stories' );
+		$dashboard      = admin_url( 'admin.php?page=googlesitekit-dashboard' );
+		$settings       = admin_url( 'admin.php?page=googlesitekit-settings' );
+
+		if ( $is_active ) {
+			if ( $is_analytics_active ) {
+				if ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
+					$analytics_link = $dashboard;
+				}
+			}
 
-		$link      = __( 'https://wordpress.org/plugins/google-site-kit/', 'web-stories' );
-		$dashboard = admin_url( 'admin.php?page=googlesitekit-dashboard' );
-		$settings  = admin_url( 'admin.php?page=googlesitekit-settings' );
-
-		if ( $is_analytics_active ) {
-			if ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
-				$link = $dashboard;
+			if ( $is_adsense_active ) {
+				if ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
+					$adsense_link = $dashboard;
+				}
 			}
-		} elseif ( $is_active ) {
+
 			if ( current_user_can( 'googlesitekit_manage_options' ) ) {
-				$link = $settings;
+				$analytics_link = $settings;
+				$adsense_link   = $settings;
 			} elseif ( current_user_can( 'googlesitekit_view_dashboard' ) ) {
-				$link = $dashboard;
+				$analytics_link = $dashboard;
+				$adsense_link   = $dashboard;
 			}
```

Comment:
This looks potentially incorrect to me: Shouldn't this logic only apply for when the respective module (Analytics / AdSense) is not active?

Reply:
I made the logic a little cleaner - https://github.com/google/web-stories-wp/pull/7622/commits/7a14a95d56da7a68f310f2f9b9e019bff1a3b408 .